src/System.Text.Json/tests/Serialization/OptionsTests.cs

```diff
@@ -25,5 +49,69 @@ public static void DefaultBufferSize()
             options.DefaultBufferSize = 1;
             Assert.Equal(1, options.DefaultBufferSize);
         }
+
+        [Fact]
+        public static void AllowTrailingCommas()
+        {
+            Assert.Throws<JsonReaderException>(() => JsonSerializer.Parse<int[]>("[1,]"));
+
+            var options = new JsonSerializerOptions();
+            options.AllowTrailingCommas = true;
+
+            int[] value = JsonSerializer.Parse<int[]>("[1,]", options);
+            Assert.Equal(1, value[0]);
+        }
+
+        [Fact]
+        public static void WriteIndented()
+        {
+            var obj = new BasicCompany();
+            obj.Initialize();
+
+            // Verify default value.
+            string json = JsonSerializer.ToString(obj);
+            Assert.DoesNotContain("\r", json);
```

Comment:
We should look for "\n" as well, especially on unix where the new line is not `\r\n`. Also, how about white space?

Reply:
Changed to space. I just need a simple test to ensure the options is passed to the writer.